### PR TITLE
v1.42.0 — Added OK, dark, and indicator badge modifiers.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Added
- OK, dark, and indicator badge modifiers.

### CODE REVIEW

File: src/scss/components/optional/_badges.scss [hunk 1]
```diff
@@ -23,9 +23,15 @@ $badge-important-color      : $white;
 $badge-important-bgColor    : $orange;
 $badge-award-color          : $purple;
 $badge-award-bgColor        : $purple--light;
+$badge-ok-color             : $white;
+$badge-ok-bgColor           : $green;
+$badge-dark-color           : $white;
```
Comment (kevinrodrigues) [inline src/scss/components/optional/_badges.scss]: Do you need to specify this one? Line 26 could be used instead?

Comment (DamianMullins) [reply src/scss/components/optional/_badges.scss]: but that variable is for the OK button?

Comment (kevinrodrigues) [reply src/scss/components/optional/_badges.scss]: OK. So a more generic one would be better so we don't end up with loads of variables pointing to the same color. but it's fine.

Comment (ashleynolan) [reply src/scss/components/optional/_badges.scss]: I'm for this as long as we don't end up with hundreds of badge types in the future (consolidation then would be good). I guess if that happens though we can optimise it then as there's still only a handful of types.

File: src/scss/components/optional/_badges.scss [hunk 2]
```diff
@@ -77,6 +83,21 @@ $badge-angled-radius        : 2px;
             color: $badge-award-color;
         }
 
+        .c-badge--ok {
```
Comment (ashleynolan) [inline src/scss/components/optional/_badges.scss]: May be worth adding a comment as to what this modifier is – `ok` is a bit vague as to the type (compared to `error` etc).  Is it used in combination with messsages?

Comment (DamianMullins) [reply src/scss/components/optional/_badges.scss]: It's only used on the item quantity indicator on menus, see attached image. I wasn't really sure what else to call it but as it's green I went with OK 😸 

Screenshot 2019-04-11 at 12 31 21

Comment (DamianMullins) [reply src/scss/components/optional/_badges.scss]: Any thoughts on what else it could be called?

Comment (DamianMullins) [reply src/scss/components/optional/_badges.scss]: I have renamed to `c-badge--success--alt` and added a description.

Comment (ashleynolan) [reply src/scss/components/optional/_badges.scss]: Sorry for delay in replying – I was away Thurs/Fri last week

That name looks good though 👍

**Pull request merged**